com.vrcfury.vrcfury/Editor/VF/Feature/ToggleBuilder.cs

```diff
@@ -69,14 +69,24 @@ public class ToggleBuilder : FeatureBuilder<Toggle> {
         return (model.name, model.usePrefixOnParam);
     }
 
+    private bool getLocalOnly() {
+        if (model.state.actions.Count() > 0) return true;
+        if (model.hasTransition) {
+            if (model.transitionStateIn.actions.Count() > 0) return true;
+            if (model.transitionStateOut.actions.Count() > 0) return true;
+        }
+        return false;
+    }
+
     [FeatureBuilderAction]
     public void Apply() {
         var fx = GetFx();
         var hasTitle = !string.IsNullOrEmpty(model.name);
         var hasIcon = model.enableIcon && model.icon?.Get() != null;
         var addMenuItem = model.addMenuItem && (hasTitle || hasIcon);
+        var networkSyncParam = addMenuItem && getLocalOnly();
 
-        var synced = true;
+        var synced = addMenuItem;

```

Comment:
I think this is a breaking change because people using a global param and then controlling that param from drivers or their own menu item would incorrectly no longer be synced.

Reply:
reverted

Reply:
un-reverted as discussed